twingate/internal/client/query/connector-create.go

```diff
@@ -4,6 +4,10 @@ type CreateConnector struct {
 	ConnectorEntityResponse `graphql:"connectorCreate(remoteNetworkId: $remoteNetworkId, name: $connectorName)"`
 }
 
+type CreateConnectorWithNotificationStatus struct {
+	ConnectorEntityResponse `graphql:"connectorCreate(remoteNetworkId: $remoteNetworkId, name: $connectorName, hasStatusNotificationsEnabled: $hasStatusNotificationsEnabled)"`
```

Comment:
does `go-graphql` support defining [defaults](https://graphql.org/learn/queries/#default-variables) here?
```
graphql:"connectorCreate(remoteNetworkId: $remoteNetworkId, name: $connectorName, hasStatusNotificationsEnabled: $hasStatusNotificationsEnabled = true)"
```

Reply:
no, it's not supported